### ISSUE TITLE
Swirl Lesson &: Matrices and Data Frame, Answer won't be accepted and can't skip

### ISSUE THREAD

**Issue by JoshuaMarshall96** [swirldev/swirl#735]
Despite completing the task as per the instructions given, I am unable to proceed. Even using the skip() function does nothing. How do I proceed?

<img width="590" alt="screen shot 2018-11-04 at 16 14 43" src="https://user-images.githubusercontent.com/37248561/47966666-ee015080-e04c-11e8-98cb-99ce8791d155.png">

**Comment by swamp-doctor** [comment]
Yes, I am having the same exact problem as JoshuaMarshall96, has anyone figured out a way to bi-pass or resolve this issue?

**Comment by alieana008** [comment]
You needed to specifically add the word data if you are going to supply the optional data argument as seen below. 
![Screenshot 2024-10-24 093146](https://github.com/user-attachments/assets/763b8948-17ae-425b-961b-bb73f0713d9f)

This is made more clear when you read the documention on it using ?matrix. It tells you the data argument is optional but if you want to use it, you need to include the word data just like you did with nrow and ncol.
![Screenshot 2024-10-24 093206](https://github.com/user-attachments/assets/0fbb73f8-334a-45ee-be59-cd971a28959e)
